pom.xml

```diff
@@ -412,6 +412,7 @@
         <module>api/metrics-api-common</module>
         <module>api/metrics-api-jaxrs</module>
         <module>api/metrics-api-jaxrs-1.1</module>
+        <module>hawkular-component</module>
```

Comment:
Le 13/10/2015 00:03, Stefan Negrea a écrit :
> What do you think about moving this module under api? Since it is meant
> for external consumption?

The api directory was created because we had two different versions of 
the REST API on top of api-common. The new module is not an "API". 
Actually I could live with the two REST module flattened in the ROOT dir

In the future - when JAX-RS 1.1 impl goes away, which should happen soon 
according to this summer's discussion :=) - I'd like to:

- merge the JAX-RS module with api-common into a single JAR module (the 
"REST" module)
- create a hawkular-metrics-standalone WAR module, with a dependency on 
the REST module
- make hawkular-metrics-standalone a true (not overlay) WAR module, with 
a dependency on the REST module